src/scss/_forms.scss

```diff
@@ -147,6 +147,20 @@ select.form-control-lg {
 }
 // scss-lint:enable QualifyingElement
 
+.form-control-success,
+.form-control-warning,
+.form-control-danger,
+.search-icon {
+  padding-right: ($input-padding-x * 3);
+  background-repeat: no-repeat;
+  background-position: center right ($input-height / 4);
+  background-size: ($input-height / 2) ($input-height / 2);
+}
+
+// Search input field icon
+.search-icon {
```

Comment:
Just for the sake of consistency with the other .form-control classes, maybe we could use a class name like `.form-control-search` instead of `.search-icon`?